### PR TITLE
adjust manifests

### PR DESCRIPTION
**Pull request by mrogers950**
Dropped the 01_serviceservingcertsigner prefix, per @smarterclayton advice. 
I also updated the image references to include the repo name and to match the tag and container name. I don't know if this is explicitly required, but it aligns with the other examples that I looked at.
/cc @openshift/sig-security @juanvallejo

### CODE REVIEW

File: manifests/image-references [hunk 1]
```diff
@@ -2,7 +2,7 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
-  - name: service-serving-cert-signer
+  - name: openshift-service-cert-signer-operator
```
Comment (smarterclayton) [inline manifests/image-references]: The name here needs to match the prefix you have when built with CI, so this was previously correct.

However, your link below is wrong, it should be whatever you have in your file:

```
name: quay.io/openshift/origin-service-serving-cert-signer
```

has to match

```
imagePullSpec: quay.io/openshift/origin-service-serving-cert-signer:latest
```

File: manifests/04_deploy.yaml [hunk 2]
```diff
@@ -18,8 +18,8 @@ spec:
     spec:
       serviceAccountName: openshift-service-cert-signer-operator
       containers:
-      - name: operator
-        image: openshift/origin-service-serving-cert-signer:latest
+      - name: openshift-service-cert-signer-operator
+        image: quay.io/openshift/origin-service-serving-cert-signer:latest
```
Comment (smarterclayton) [inline manifests/04_deploy.yaml]: This is the string that should show up in `image-references` under the tag.  You don't need to change the container name.

**Comment by mrogers950** [comment]
@smarterclayton thanks, updated.

**Comment by abhinavdahiya** [comment]
Also seems like service account is next in order to deployment, The deployment will not complete rollout with out existing service account and ClusterVersionOperator waits for rollout to complete for deployment as `done criteria`. Do you mind changing the order to serviceaccount then deployment.

**Comment by mrogers950** [comment]
@abhinavdahiya done

**Comment by mrogers950** [comment]
@smarterclayton @abhinavdahiya are the manifest filenames here still correct or do I need to prepend a `0000_01` to them? I noticed this might have changed per your comment at https://github.com/openshift/cluster-openshift-apiserver-operator/pull/15#issuecomment-425481195

**Comment by smarterclayton** [comment]
Do any components depend on you being installed before they are installed?  That is the only reason you would be allowed to use the run level notation.

**Comment by juanvallejo** [comment]
@smarterclayton 
> Do any components depend on you being installed before they are installed? That is the only reason you would be allowed to use the run level notation.

I believe that the openshift-apiserver would depend on this operator. cc @deads2k

File: manifests/0000_05_serviceservingcertsigner_05_deploy.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: openshift-service-cert-signer-operator
       containers:
       - name: operator
-        image: openshift/origin-service-serving-cert-signer:latest
+        image: quay.io/openshift/origin-service-serving-cert-signer:latest
```
Comment (smarterclayton) [inline manifests/0000_05_serviceservingcertsigner_05_deploy.yaml]: Your filenames need to be dash separated - `0000_05_service-serving-cert-signer_05_deploy.yaml`.  That makes it consistent with everyone elses.

**Pull request merged**